macros/audit.sql

```diff
@@ -1,37 +1,54 @@
+{% macro get_audit_schema() %}
+
+    {# if the get_audit_schema macro exists in the base project use that #}
+    {% if context.get(project_name, {}).get('get_audit_schema') %}
+        {{ return(context[project_name].get_audit_schema()) }}
+    {% else %}
+        {{ return(target.schema~'_meta') }}
+    {% endif %}
+
+{% endmacro %}
+
 {% macro get_audit_relation() %}
+    {%- set audit_schema=logging.get_audit_schema() -%}
+
     {%- set audit_table =
         api.Relation.create(
+            database=target.database,
+            schema=audit_schema,
```

Comment:
@bobzoller I think we want these to remain static

Reply:
before I use this PR in production, I'll be adding this macro to legacy, which (if I'm grokking this codebase right) will make the schema static:
```
{% macro get_audit_schema() %}
  {{ return('dbt_meta') }}
{% endmacro %
```
seem right?

Reply:
https://github.com/fishtown-analytics/dbt-event-logging#customising-audit-schema